twilight-model/src/guild/onboarding/option.rs

```diff
@@ -0,0 +1,209 @@
+use crate::{
+    guild::Emoji,
+    id::{
+        marker::{ChannelMarker, EmojiMarker, OnboardingPromptOptionMarker, RoleMarker},
+        Id,
+    },
+};
+use serde::{Deserialize, Serialize};
+use std::fmt::{Display, Formatter};
+
+/// An emoji for a guild onboarding prompt.
+/// This is used instead [`Emoji`] as both it's id and name can be `null` in prompt options.
+///
+/// [`Emoji`]: crate::guild::Emoji
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct OnboardingPromptEmoji {
```

Comment:
Reinvention of [`ReactionType`](https://docs.rs/twilight-model/latest/twilight_model/channel/message/enum.ReactionType.html)? The doc PR was maybe updated, the given examples don't include ID and name being `null` at the same time.